### ISSUE TITLE
use pretrained weights from Swin transformer

### ISSUE THREAD

**Issue by kiashann** [JingyunLiang/SwinIR#86]
Hi, did you used pretrained weights from Swin transformer?
Could you please share your training files that we understand about this?
Which type of Swin transformer (large, small,...) did you use for training?
@JingyunLiang

**Comment by JingyunLiang** [comment]
No, the model is trained from scratch. See https://github.com/cszn/KAIR/blob/master/docs/README_SwinIR.md for training codes.

**Comment by kiashann** [comment]
> No, the model is trained from scratch. See https://github.com/cszn/KAIR/blob/master/docs/README_SwinIR.md for training codes.

It is possible to use pretrained weights?
Why you didn't share train.py file?

**Comment by JingyunLiang** [comment]
The architecture is different. Besides, there is no much meaning to use weights pretrained on classfication.

The train.py is provided. Please refer to the Readme.
